diff --git a/packages/PyTrilinos/src/CMakeLists.txt b/packages/PyTrilinos/src/CMakeLists.txt
index c7d7cf1..d3e0cee 100644
--- a/packages/PyTrilinos/src/CMakeLists.txt
+++ b/packages/PyTrilinos/src/CMakeLists.txt
@@ -191,12 +191,6 @@ TRIBITS_ADD_LIBRARY(pytrilinos
   SOURCES ${SOURCES}
   )
 
-# Link against the Python library to prevent undefined
-# symbols for shared builds with -Wl,--no-undefined.
-TARGET_LINK_LIBRARIES(pytrilinos
-  ${PYTHON_LIBRARIES}
-  )
-
 SET_TARGET_PROPERTIES(pytrilinos
   PROPERTIES LINK_FLAGS "${EXTRA_LINK_ARGS}"
   )
