diff --git a/packages/galeri/src/iohb.cpp b/packages/galeri/src/iohb.cpp
index 6e10a44..fdbddec 100644
--- a/packages/galeri/src/iohb.cpp
+++ b/packages/galeri/src/iohb.cpp
@@ -773,7 +773,7 @@ int writeHB_mat_double(const char* filename, int M, int N,
        }
     } else out_file = stdout;
 
-    if ( Ptrfmt == NULL ) Ptrfmt = "(8I10)";
+    if ( Ptrfmt == NULL ) strcpy(Ptrfmt, "(8I10)");
     ParseIfmt(Ptrfmt,&Ptrperline,&Ptrwidth);
     sprintf(pformat,"%%%dd",Ptrwidth);
     ptrcrd = (N+1)/Ptrperline;
@@ -786,7 +786,7 @@ int writeHB_mat_double(const char* filename, int M, int N,
     if ( nz%Indperline != 0) indcrd++;
 
     if ( Type[0] != 'P' ) {          /* Skip if pattern only  */
-      if ( Valfmt == NULL ) Valfmt = "(4E20.13)";
+      if ( Valfmt == NULL ) strcpy(Valfmt, "(4E20.13)");
       ParseRfmt(Valfmt,&Valperline,&Valwidth,&Valprec,&Valflag);
       if (Valflag == 'D') *strchr(Valfmt,'D') = 'E';
       if (Valflag == 'F')
@@ -1356,7 +1356,7 @@ int writeHB_mat_char(const char* filename, int M, int N,
        }
     } else out_file = stdout;
 
-    if ( Ptrfmt == NULL ) Ptrfmt = "(8I10)";
+    if ( Ptrfmt == NULL ) strcpy(Ptrfmt, "(8I10)");
     ParseIfmt(Ptrfmt,&Ptrperline,&Ptrwidth);
     sprintf(pformat,"%%%dd",Ptrwidth);
 
@@ -1365,7 +1365,7 @@ int writeHB_mat_char(const char* filename, int M, int N,
     sprintf(iformat,"%%%dd",Indwidth);
 
     if ( Type[0] != 'P' ) {          /* Skip if pattern only  */
-      if ( Valfmt == NULL ) Valfmt = "(4E20.13)";
+      if ( Valfmt == NULL ) strcpy(Valfmt, "(4E20.13)");
       ParseRfmt(Valfmt,&Valperline,&Valwidth,&Valprec,&Valflag);
       sprintf(vformat,"%%%ds",Valwidth);
     }
