diff --git a/cmake/TPLs/FindTPLNetcdf.cmake b/cmake/TPLs/FindTPLNetcdf.cmake
index 814d289..87121ee 100644
--- a/cmake/TPLs/FindTPLNetcdf.cmake
+++ b/cmake/TPLs/FindTPLNetcdf.cmake
@@ -53,8 +53,18 @@
 # ************************************************************************
 # @HEADER
 
+FIND_PACKAGE(netCDF)
 
-TRIBITS_TPL_FIND_INCLUDE_DIRS_AND_LIBRARIES( Netcdf
-  REQUIRED_HEADERS netcdf.h
-  REQUIRED_LIBS_NAMES netcdf
-  )
+IF(netCDF_LIBRARIES)
+  SET(TPL_Netcdf_INCLUDE_DIRS ${netCDF_INCLUDE_DIR} CACHE PATH
+    "Netcdf include dirs")
+  SET(TPL_Netcdf_LIBRARIES ${netCDF_LIBRARIES} CACHE FILEPATH
+    "NETCDF libraries")
+  SET(TPL_Netcdf_LIBRARY_DIRS ${netCDF_LIB_DIR} CACHE PATH
+    "NETCDF library dirs")
+ELSE()
+  TRIBITS_TPL_FIND_INCLUDE_DIRS_AND_LIBRARIES( Netcdf
+    REQUIRED_HEADERS netcdf.h
+    REQUIRED_LIBS_NAMES netcdf
+    )
+ENDIF()
